Products/CMFPlone/profiles/default/metadata.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>5020</version>
+  <version>5021</version>
```

Comment:
@b4oshany The correct way to get this updated is probably to rebase this branch on top of the current 5.0.x.